public/js/office.web.js

```diff
@@ -175,13 +166,23 @@ $(() => {
   }
 
   function getLastRoom(matrixProfile){
-  	var lastRoom = localStorage.getItem(`last_room${matrixProfile.loadStoredProfile().id}`);
+
+    var lastRoom = getUrlRoom();
+
+    if(lastRoom==null || lastRoom==undefined || lastRoom== "undefined"){
```

Comment:
This logic seems to be duplicated, what about move it to a function and use this function here and in line 176?

Maybe, you could call the function as `lastRoomIsInvalid()`.